c++/src/kj/async-io-win32.c++

```diff
@@ -1114,6 +1114,16 @@ private:
 Promise<void> AsyncInputStream::read(void* buffer, size_t bytes) {
   return read(buffer, bytes, bytes).then([](size_t) {});
 }
+Promise<size_t> AsyncInputStream::read(void* buffer, size_t minBytes, size_t maxBytes) {
```

Comment:
This change is no longer needed at master -- I moved the default implementations of virtual functions into a common file for Unix and Windows.